src/pcsaft/eos/polar.rs

```diff
@@ -99,39 +99,20 @@ impl MeanSegmentNumbers {
             Multipole::Quadrupole => (parameters.nquadpole, &parameters.quadpole_comp),
         };
 
```

Comment:
the old implementation is unnecessarily complicated for a negligible benefit in execution time.